pyedflib/edfreader.py

```diff
@@ -124,60 +157,69 @@ def _get_float(self, v):
                 result[i] = float(v[i])
         return result
 
-    def _convert_string(self,s):
+    # FIX: Seems s must always be bytes because string has no decode method
+    def _convert_string(self, s: Union[bytes, str]) -> str:
```

Comment:
```suggestion
    def _convert_string(self, s: Union[bytes]) -> str:
```

how was the `str` suggested? Is the method called with `str` somewhere?